packages/next/test/fixtures/00-app-dir-actions-streaming-index-handling/package.json

```diff
@@ -1,11 +1,10 @@
 {
   "dependencies": {
-    "next": "canary",
+    "next": "15.2.0-canary.33",
```

Comment:
This is the point of the whole PR. These `canary` tests fail somewhat often for various reasons, blocking other work in this repository.

By pinning these versions, we isolate some of these failures to only a single PR that updates next versions (from the `utils/update-next.js` script).